repos/system_upgrade/common/actors/checksaphana/actor.py

```diff
@@ -12,17 +14,52 @@ class CheckSapHana(Actor):
     If the upgrade flavour is 'default' no checks are being executed.
 
     The following checks are executed:
-    - If this system is _NOT_ running on x86_64, the upgrade is inhibited.
-    - If SAP HANA 1 has been detected on the system the upgrade is inhibited since it is not supported on RHEL8.
+    - If the major target release is 8, and this system is _NOT_ running on x86_64, the upgrade is inhibited.
+    - If the major target release is 9, and this system is _NOT_ running on x86_64 or ppc64le,
```

Comment:
I don't see a change for the support of ppc64le - Did you miss that? Because it's not changed here in this PR but you document it that way. And also you say it's also for RHEL9, which most definitely needs a code change where the architecture is checked for SAP Hana (If it is at all)

Reply:
ppc64le support was added by #PR1042 , but this docstring was not updated.